### DISCUSSION TITLE
Lesson 6 : Brownie Fund Me

### DISCUSSION THREAD

**Discussion by wilonweb** [smartcontractkit/full-blockchain-solidity-course-py#1526] [Q&A] [answered]
Hello,
at 05.44.05 from the tuto video, the teacher add a function named `getEntraceFee`
And the tutorial say copy the function from the gitHub repo. 
But i don't find the function in the gitHub repo. 
Where can i find it please.

**Comment by cromewar** [comment, marked as answer]
Hello @NinjaTurtles-cloud 
Here you have: https://github.com/PatrickAlphaC/brownie_fund_me/blob/main/contracts/FundMe.sol
